### PR TITLE
2.3.1

### PR DESCRIPTION
**Pull request by jinglitw**
# API Pull Request
## VERSION Number
- 2.3.1
## Ticket url
- https://chillibean.atlassian.net/browse/PVA-931
## Changes Made
- Added AssetShort schema
- Adjusted study procedure assets move schema to match current state
## Checklist
Please verify this checklist so that you are sure this is ready for review.

- [ ]  The only file that has been changed is schema.json - unless you needed to add some new functionality (if so, please update generator repo too)
- [ ]  Updated package.json & schema.json versions - they need to be the same
- [ ]  All objects have a properites list, required array and additionalProperties set to false

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,19 +68,20 @@ Here's an example of the development workflow:
...
+7. Run **`npm run bundle`** to bundle the **`/public/schema.json`** to generate **`/public/fullSchema.json`**.
+8. Run **`npm version 1.1.10-alpha.0`**.
+9. Push to GitHub and create a pull request.
+10. A pre-release is generated, the Client SDK is rebuilt, and the package is published.
+11. Update your service with the new SDK to test the alpha version.
+12. If everything works, get approval to merge your branch into the release branch **`1.0.10`**.
+13. Once ready in **`1.1.10`**, get approval to merge into **`main`**.
+14. Merging into **`main`** generates a release, updates the Client SDK, and publishes a release package.
+15. Update your service with the new **`1.1.10`** version.
+16. Repeat!
```
Comment (robertozmc) [inline README.md]: why this has to be changed?

File: README.md [hunk 2]
```diff
@@ -131,7 +132,7 @@ You can run **`npm run lint`** to check the contents of **`/public/schema.json`*
 
 ## **Bundling**
 
-You can run **`npm run bunlde`** to bundle the **`/public/schema.json`** file into the full schema that outputs to **`/public/fullSchema.json`**.
+You can run **`npm run bundle`** to bundle the **`/public/schema.json`** file into the full schema that outputs to **`/public/fullSchema.json`**.
```
Comment (robertozmc) [inline README.md]: good catch

File: public/refs/components/schemas/Video.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "type": "object",
+  "description": "The video transcode",
```
Comment (robertozmc) [inline public/refs/components/schemas/Video.json]: I feel this schema file should be also named `VideoTranscode.json`

File: public/refs/components/schemas/Video.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  "properties": {
+    "url": {
+      "type": "string",
+      "nullable": true,
+      "description": "The asset  video url",
```
Comment (robertozmc) [inline public/refs/components/schemas/Video.json]:
```suggestion
      "description": "The asset video url",
```

File: public/refs/components/schemas/Video.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      "example": "http://path-to-video"
```
Comment (robertozmc) [inline public/refs/components/schemas/Video.json]: don't you want to use the same url as in `Asset.json` schema, but with mp4 extension?

File: public/refs/components/schemas/Video.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "progress": {
+      "type": "string",
+      "nullable": true,
+      "description": "The asset video progress",
```
Comment (robertozmc) [inline public/refs/components/schemas/Video.json]: this is transcoding progress

File: public/refs/components/schemas/Video.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+      "example": "100"
+    },
+    "status": {
+      "type": "string",
+      "nullable": true,
+      "description": "The asset video status",
```
Comment (robertozmc) [inline public/refs/components/schemas/Video.json]: this is transcoding status

File: public/refs/components/schemas/Video.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
+{
+  "type": "object",
+  "description": "The video transcode",
+  "properties": {
```
Comment (robertozmc) [inline public/refs/components/schemas/Video.json]: I wonder if we could order properties in alphabetical order?

**Review by robertozmc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jinglitw) [reply README.md]: Add one missing step => 7. Run **`npm run bundle`** to bundle the **`/public/schema.json`** to generate **`/public/fullSchema.json`**.

Another tiny thing => New feature version should be `1.1.10` not `1.0.11`

Comment (robertozmc) [reply README.md]: I missed that one, thanks 😊

**Review by robertozmc — APPROVED** [review]

**Pull request merged**